### DISCUSSION TITLE
Has this project been abandoned?

### DISCUSSION THREAD

**Discussion by doftheworld** [chuot/rdio-scanner#382] [General]
This a great program, but with no updates in more than a year I am wondering if it has been abandoned? I am not complaining, just trying to figure out where I should focus my time in the future.

**Comment by chuot** [comment]
No, it hasn't. It's just that I have to deal with 2 jobs at the same time, because of a divorce with a partner in life for over twenty years and a business partner for 10 years, kids under majority, and life hasn't been easy on me lately.

But rest assured, I'm working hard to bring you the next big thing. Bare with me, you won't be disappointed!

Regards

**Reply by doftheworld** [reply]
I am sorry to hear that you've had so much to deal with. That really sounds like more than I could handle. I hope you don't feel like this was me calling you out - I was just trying to figure out where I should focus my attention. I have full faith in you as this is by far the best project out there.

**Reply by chuot** [reply]
I love this project, and I'm putting my heart into it. I learned so much from it and do want to pursue on this road. Thanks for liking my project despite the poor feedback I gave to this community along the last ten months or so.

**Reply by TheTypicalHam** [reply]
Glad to see this. I recently discovered it and it has great potential! I know there are some smart people out there in this community that would probably be willing to chip in some help if you look, but nothing I can do to help except donate at this time.

**Comment by doftheworld** [comment]
> I love this project, and I'm putting my heart into it. I learned so much from it and do want to pursue on this road. Thanks for liking my project despite the poor feedback I gave to this community along the last ten months or so.
I have been working on a project that is (extremely) losely connected and I know how it feels to take pride in it. I am ignorant about programing, except for what is connected to my own project. I hope in the future I can learn more about this and give input, but for the time being, the best I can provide is feedback.
